axelor-purchase/src/main/resources/i18n/messages_fr.csv

```diff
@@ -65,6 +65,7 @@
 "Draft","Brouillon",,
 "Estim. delivery date","Date de livraison estimée",,
 "Financial terms","Termes financiers",,
+"Finish manually purchase order","Terminer manuellement les commandes fournisseurs",,
```

Comment:
This term is not good. This message appears on the button on each purchase order, so just put "Finish" (Terminer) or "Finish the order" (Terminer la commande)